packages/dds/matrix/src/matrix.ts

```diff
@@ -435,6 +437,28 @@ export class SharedMatrix<T extends Serializable = Serializable>
         return tree;
     }
 
+    /**
+     * Returns the GC data for this SharedMatrix. All the IFluidHandle's stored in the cells represent routes to other
+     * objects.
+     */
+    protected getGCDataCore(): IGarbageCollectionData {
+        // Create a SummarySerializer and use it to serialize all the cells. It keeps track of all IFluidHandles that it
+        // serializes.
```

Comment:
what about this.rows and this.cols? can those not have handles?

Reply:
Yes, that's my understanding. @DLehenbauer to confirm.

Reply:
Confirmed: The MergeTree segments contain a SharedMatrix notion of "handle to physical storage", which is just an integer.